app/controllers/certificates/a_level_controller.rb

```diff
@@ -0,0 +1,85 @@
+module Certificates
+  class ALevelController < ApplicationController
+    layout 'full-width'
+    before_action :authenticate_user!
+    before_action :find_programme, only: %i[show complete pending]
+    before_action :user_enrolled?, only: %i[show complete pending]
+    before_action :user_programme_enrolment_pending?, only: %i[show complete]
+    after_action :discourage_caching
+
+    def show
+      return redirect_to complete_a_level_path if @programme.user_completed?(current_user)
+
+      assign_achievements
+      @professional_development_groups = @programme.programme_activity_groupings.not_community.order(:sort_key)
```

Comment:
Should there be a list of enrolled+completed courses that aren't listed in the programme activity groupings?

Reply:
No, it doesn't look like we currently show completed courses that are not recommended